test/e2e/suite/command/verify.go

```diff
@@ -121,4 +121,32 @@ var _ = Describe("notation verify", func() {
 				MatchErrKeyWords(expectedErrMsg)
 		})
 	})
+
+	It("with TLS by digest", func() {
+		HostInGithubAction(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			notation.Exec("sign", artifact.DomainReferenceWithDigest()).
+				MatchKeyWords(SignSuccessfully)
+
+			notation.Exec("verify", "-d", artifact.DomainReferenceWithDigest(), "-v").
```

Comment:
nit:
For this one, since we already turned on "-d", we can remove "-v".
```suggestion
			notation.Exec("verify", "-d", artifact.DomainReferenceWithDigest()).
```

Reply:
Removed.